### PR TITLE
Implement complete-plan command V1.2

### LINKED ISSUE #47: Implement complete-plan command
- Implement complete-plan command
- Implement testcases

### PR DESCRIPTION
**Pull request by GohTengFong**
- Implemented complete-plan command along with associated classes
- Implemented testcases

### CODE REVIEW

File: src/main/java/seedu/address/model/Model.java [hunk 1]
```diff
@@ -121,7 +125,7 @@ public interface Model {
      */
     void updateFilteredPersonList(Predicate<Person> predicate);
 
-    /**
```
Comment (Elijah5399) [inline src/main/java/seedu/address/model/Model.java]: Is this a normal comment or JavaDoc comment?

File: src/main/java/seedu/address/model/plan/Plan.java [hunk 2]
```diff
@@ -69,6 +72,7 @@ public String toString() {
         String friendName = friend.getName().toString();
         String dateTimeString = this.dateTime.toString();
         String planNameString = this.planName.toString();
+        String planCompleteString = this.isComplete.toString();
```
Comment (Elijah5399) [inline src/main/java/seedu/address/model/plan/Plan.java:75]: planCompleteString can be added to the returned string

**Review by Elijah5399 — COMMENTED** [review]
Have picked out certain nits to be fixed. Other than that, LGTM once the remaining tests pass.

File: src/main/java/seedu/address/model/Model.java (hunk 1 above)
Comment (GohTengFong) [reply src/main/java/seedu/address/model/Model.java]: rectified in latest push

**Review by Elijah5399 — APPROVED** [review]
LGTM!

**Pull request merged**